src/transformers/models/convnextv2/convert_convnextv2_to_pytorch.py

```diff
@@ -99,6 +99,10 @@ def rename_key(name):
     if "stages" in name and "downsampling_layer" not in name:
         # stages.0.0. for instance should be renamed to stages.0.layers.0.
         name = name[: len("stages.0")] + ".layers" + name[len("stages.0") :]
+    if "gamma" in name:
+        name = name.replace("gamma", "weight")
+    if "beta" in name:
+        name = name.replace("beta", "bias")
```

Comment:
You don't need to do it since `from_pretrained` will do it anyway. But it doesn't hurt I guess :sweat_smile:

Reply:
Ah true, I renamed them differently at first and then switched to weight and bias to avoid updating the checkpoints